less/_defaults/_spacing-containers.less

```diff
@@ -19,7 +19,7 @@
 @menu-padding-hoz: 0;
 
 @menu-header-padding-ver: 2rem;
-@menu-header-padding-hoz: .5rem;
+@menu-header-padding-hoz: 1rem;
```

Comment:
Bumping up the menu header means this doesn't align with the menu items below. Not sure if we would want to bump the [menu item margin](https://github.com/adaptlearning/adapt-contrib-vanilla/blob/aac3a2bdade9398bc9cf353dd40b64f5235096dd/less/plugins/adapt-contrib-boxmenu/boxMenuItem.less#L3) for consistency?
<img width="576" alt="header_and_item_alignment" src="https://github.com/adaptlearning/adapt-contrib-vanilla/assets/7045330/40999381-227f-4051-94dd-739c7ed313b1">

Reply:
Agreed

Reply:
Although, I think bumping the menu item margin could be more problematic with scaling/overrun issues. Particularly if you're using column classes etc where your items are rendered at smaller widths. Maybe revert menu header back to `0.5rem` rather than amending the menu item margin? Happy to go with whichever people feel is less of an issue.

Reply:
The menu item margin in hard-coded in the theme. I think the theme needs an overall modernisation and rationalisation. I'll create a separate issue for this, I just need to do a bit more prep work. In the meantime lest try and tidy these bits up. I'll have a at any issues this may cause now.

Reply:
@kirsty-hames, we can't increate the item spacing to effect the layout margins as that increases the gutter between the items. Suggest adding a 0.5 rem margin to .boxmenu__item-container to compensate for now. Then we can do a better review holistically when I create a new issue. 

@guywillis / @kirsty-hames - what do you think?

Reply:
> @kirsty-hames, we can't increase the item spacing to affect the layout margins as that increases the gutter between the items. Suggest adding a 0.5 rem margin to .boxmenu__item-container to compensate for now. Then we can do a better review holistically when I create a new issue.
>
Yep I'd rather not increase the gutter. I'm happy to go with your suggestion thanks.

Reply:
There isn't really a perfect way of doing this currently without a bit of a rethink. In my opinion, the best solution is to add an extra var -> @menu-item-container-padding-hoz: 1rem;
 then update that item container with this:
 
```css
   &__item-container {
    margin: 0 -@menu-item-padding-hoz;
    padding: 0 @menu-item-container-padding-hoz;
  }
```
 
 I'll also add extra vars of these and update box menu also:
 
@menu-item-padding-ver: 1rem;
@menu-item-padding-hoz: 0.5rem;

Obviously, boxmenu would require these updates to the theme. Is there a formal way where we document this?